app/models/planet.py

```diff
@@ -0,0 +1,25 @@
+from app import db
+
+class Planet(db.Model):
+    id = db.Column(db.Integer, primary_key = True, autoincrement= True)
+    name = db.Column(db.String)
+    description = db.Column(db.String)
+    color = db.Column(db.String)
+
+    def to_json(self):
+        return {"id": self.id,
+                "name": self.name,
+                "description": self.description,
+                "color": self.color
+                }
+    def update_planet(self,request_body):
+        self.name = request_body["name"]
+        self.description =request_body["description"]
+        self.color= request_body["color"]
+    
+    @classmethod
+    def create_planet(cls, request_body):
+        new_planet = cls(name = request_body["name"],
+                        description = request_body["description"], 
+                        color = request_body["color"])
+        return new_planet
```

Comment:
Nice refactor here to keep your routes concise.

This comment applies to your update_planet() and create_planet() methods:

How would you add in validation to make sure that all the required fields are sent in the request to your server?

For example, if someone sent a POST request but left off color, then line 24 would throw KeyError that it couldn't find color.

it would be nice to handle the error and return a message so that the client knows their request was invalid and they need to include color. Something to think about for Task List.